Px.Utils/ModelBuilders/MatrixMetadataBuilder.cs

```diff
@@ -91,12 +91,12 @@ private MatrixMetadata BuildFromentries(Dictionary<MetadataEntryKey, string> ent
             string stubKey = _pxFileSyntaxConf.Tokens.KeyWords.StubDimensions;
             IEnumerable<MultilanguageString> stubDimensionNames = TryGetAndRemoveProperty(entries, stubKey, langs, out MetaProperty? maybeStub)
                 ? maybeStub.ValueAsListOfMultilanguageStrings(langs.DefaultLanguage, _listSeparator, _stringDelimeter)
-                : throw new ArgumentException("Stub variable names not found in metadata");
```

Comment:
STUB and HEADING keywords are mandatory, if they are missing an exception should be thrown and handled in the calling function.

Reply:
Change reverted